README.md

```diff
@@ -226,7 +229,7 @@ Frontend set up steps:
 
 1) Navigate into the **frontend** directory
     ```
-    cd server/
+    cd fronts
```

Comment:
Kindly pull updates &merger with master branch (* The directory is named `frontend`)